### ISSUE TITLE
Navigation problem (return key)

### ISSUE THREAD

**Issue by Bindou01** [jurialmunkey/skin.arctic.fuse#902]
### Skin section

Widgets

### Current Behavior

Jurial, the return function has never been so close to perfection, intuitive and reliable.  However, I still encounter two last small bugs.  Indeed, when I am on the last Widget of my first category, clicking on return does not take me back to the top, it has no effect.  While on all Widgets it works perfectly, it brings you back to the list of categories.  Finally the second bug: when I am in a Hub made up of Widgets for the TMDb Helper series, if I enter a Season, and I want to leave it, the return click not only leaves the season folder, but takes us back to the top  Widgets (on the list of categories) directly.  Sorry to bore you with this, I know you've been working on the navigation of the back function for a while.  But honestly, I think you're getting to the end.

### Expected Behavior

..

### Steps To Reproduce

..

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
I can only reproduce the second issue. I cannot reproduce the first issue. 

Also please follow the rules and report separate bugs as separate issues. It is the very first checkbox to tick "I have made a separate issue for each bug I am reporting".

Reporting multiple bugs in one issue thread makes it harder for me to address.

**Comment by jurialmunkey** [comment]
The hub return issue is fixed. Please report your other issue as a separate thread as per the rules.

**Comment by Bindou01** [comment]
> The hub return issue is fixed (second issue). Please report your other issue (first issue) as a separate thread as per the rules.

Concerning the first bug, since my last reboot, I can no longer reproduce it either for the moment.  If this happens again I will create a new, separate report.

 Thank you very much for resolving the second bug so quickly.  It works completely perfectly for now 🙏

 Honestly, Fuse seems perfectly functional to me at this stage, it is so intelligently designed and it is so aesthetic, that I think it would one day deserve to become the official Kodi skin.  When the last little details are resolved.  I think it would be worth pushing in this direction.  Fuse literally unblocks Kodi!

 Thank you for sharing your passionate and professional work with us.  Thank you for listening to your community.  And then, you have attention to detail, I love that.  That's what makes the difference.

**Comment by Bindou01** [comment]
@jurialmunkey 
Small Jurial correction, problem 2, is not completely resolved.  It only works if you focus on the thumbnails.  but we are positioned on the lower tab "Season 1" nothing happens when you press the return key.
